docs/models.txt

```diff
@@ -685,10 +686,11 @@ Interface mediafile {
     mimetype: string;
     pdf_information: JSON;
     create_timestamp: datetime;
-    path: string; // Note: calcuated
-    inherited_access_group_ids: boolean | number[]; // Note: calculated and no
-    // reverse-relation for the group ids.
+    path: string; // Note: calculated
 
+    is_public: boolean;
```

Comment:
I see why it's needed, but this field is calculated, right? If so, please add a note about it (I can smell confusion on both side about not allowing to write to this field ;) )

Reply:
Yes, right. Also renamed it after looking again at the OS3 code